File: iniciante/C++/1060-Positive_numbers.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
    float num;
    int i, pos=0;
    for (i=0; i<6; i++) {
        cin >> num;
        if (num > 0) {
            pos += 1;
        }
    }
    cout << pos << " valores positivos\n";
    return 0;
}

File: iniciante/Dart/1060-Positive_numbers.dart
import 'dart:io';

void main() {
  int cont = 0;
  for (int i = 0; i < 6; i++) {
    // Remove the ! before submitting the solution...
    var Num = double.parse(stdin.readLineSync()!);
    if (Num > 0) cont += 1;
  }
  print("${cont} valores positivos");
}
